packages/@react-aria/calendar/src/useCalendarBase.ts

```diff
@@ -39,36 +39,42 @@ export function useCalendarBase(props: CalendarPropsBase & DOMProps, state: Cale
   // Kinda hacky, but it's the easiest way to get the id from the cell.
   calendarIds.set(state, calendarId);
 
+  let focusFocusedDateCell = useCallback(() => {
+    const focusedCell = calendarBody.current.querySelector(`#${getCellId(state.focusedDate, calendarId)}`);
+    if (focusedCell && focusedCell !== document.activeElement) {
+      focusedCell.focus();
+    }
+  }, [state.focusedDate, calendarId]);
+
   useEffect(() => {
     // focus the calendar body when mounting
     if (autoFocus) {
-      calendarBody.current.focus();
+      focusFocusedDateCell();
     }
   }, [autoFocus]);
```

Comment:
This is going to throw a hooks lint warning about useEffect and its dependencies, can you add a comment explaining why you aren't including `focusFocusedDateCell` in here as well?
Also maybe lint disable it?

Reply:
I can lint disable it. The problem when I include focusFocusedDateCell is that autoFocus executes whenever the month changes, which we don't want when the user is navigating using the Next or Previous Month button.

Reply:
lets add that as a comment in the code